documentation/markdown_and_code_dev.md

```diff
@@ -9,44 +9,67 @@ These include {term}`Jupyter Notebook`s, Linux {term}`command-line <terminal>` a
 
 A Jupyter Notebook is a feature-rich application supporting data exploration and code development.
 The notebook documents may be in either `.ipynb` or `.md` (Markdown) format.
-The Fornax Tutorial Notebooks are in `.md` format following the standard practice of the Scientific Python ecosystem.
+The Fornax Tutorial Notebooks are in `.md` format ([MyST](https://mystmd.org/) Markdown flavor) following the standard practice of the Scientific Python ecosystem.
 While many astronomers are familiar with the `.ipynb` format, we chose Markdown for its superior readability, ease of version control through diffing, simpler testing workflows, and smooth rendering to HTML.
 To read more about the differences between `.md` and `.ipynb`, see the [MyST Markdown documentation](https://mystmd.org/guide/md-vs-ipynb).
 
 ### JupyterLab Markdown Dependencies
 
 On the Fornax Science Console, all required dependencies for using Markdown in JupyterLab are pre-installed.
+
 To use Markdown notebooks in JupyterLab outside of the Fornax Science Console, ensure the following tools are installed:
 
 -   [`jupytext`](https://github.com/mwouts/jupytext) Python library
 -   [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) JupyterLab extension
 
+(open-md-in-jupyterlab)=
 ### Opening `.md` Files in JupyterLab
 
-1.  Navigate to the `.md` file in the JupyterLab file browser.
-2.  Right-click the file, select **Open With**, and choose **Notebook** or **Jupytext Notebook**.
-3.  The Markdown file will open in an interactive notebook interface with cells and execution capabilities—just like a traditional `.ipynb` notebook.
+In the Fornax Science Console, you can simply double click on a `.md` file in the file browser.
+It will open in an interactive notebook interface with cells and execution capabilities — just like a traditional `.ipynb` notebook.
+If you need to save the notebook output, pair it with a `.ipynb` document before running it by selecting: `File → Jupytext → Pair Notebook with ipynb document`.
```

Comment:
I would prefer not to have any docs on pairing. Pairing in my experience does more confusion and mess than purpose it solves (the ecosystem came a long way from the time when it was necessary)

Reply:
Alternative: have a proper data management plan -- data, it it has any value, should not be stored in a json file as binary blob -- but been save out to a file.

(but also, as long as the session is active I suppose it all sees the outputs. those are stored in temp locations, it's just the notebook won't pick up those by default. And now I'm not sure if that can be pulled back up as from a cache later on, just like we do for the rendering.

Still -- this is a documentation for the notebook that we have and for those I still don't recommend the conversions -- they will be overwritten anyway in a new login to the platform.

Reply:
This page is not specifically about the notebooks we've developed, it's about any code the users want to develop or work with on the platform. Regardless of who wrote a notebook, there are legitimate reasons to want to save an executed version. We need to give users enough information to be able to do that. If a better suggestion for how to do that comes along then we can update these instructions.